content/en/agent/guide/datadog-disaster-recovery.md

```diff
@@ -0,0 +1,284 @@
+curl -X GET "https://api.<DD-SITE-PARAMETER>/api/v1/org/<PUBLIC-ID>" \
+-H "Accept: application/json" \
+-H "DD-API-KEY: ${PRIMARY_DD_API_KEY}" \
+-H "DD-APPLICATION-KEY: ${PRIMARY_DD_APP_KEY}"
+
+# Run this command to verify the Public ID for your DDR site replacing the `<DD-SITE-PARAMETER>` with the correct value
+curl -X GET "https://api.<DD-SITE-PARAMETER>/api/v1/org/<PUBLIC-ID>" \
+-H "Accept: application/json" \
+-H "DD-API-KEY: ${DDR_DD_API_KEY}" \
+-H "DD-APPLICATION-KEY: ${DDR_DD_APP_KEY}"
+
+```
+{{% /collapse-content %}}
+
+
+{{% collapse-content title=" 4. Link the DDR org to the primary org" level="h5" %}}
+After the Datadog team has completed the configuration of the designated orgs and you have confirmed the public IDs for your orgs, you can now link them. For security reasons, Datadog is unable to link the orgs on your behalf. 
+
+To link your primary and DDR orgs, run these commands:
+
+```shell
+export PRIMARY_DD_API_KEY=<PRIMARY_ORG_API_KEY>
+export PRIMARY_DD_APP_KEY=<PRIMARY_ORG_APP_KEY>
+export PRIMARY_DD_API_URL=<PRIMARY_ORG_API_SITE>
+
+export DDR_ORG_ID=<DDR_ORG_PUBLIC_ID>
+export PRIMARY_ORG_ID=<PRIMARY_ORG_PUBLIC_ID>
+export USER_EMAIL=<USER_EMAIL>
+export CONNECTION='{"data":{"id":"'${PRIMARY_ORG_ID}'","type":"hamr_org_connections","attributes":{"TargetOrgUuid":"'${DDR_ORG_ID}'","HamrStatus":1,"ModifiedBy":"'${USER_EMAIL}'", "IsPrimary":true}}}'
+
+curl -v -H "Content-Type: application/json" -H 
+"dd-api-key:${PRIMARY_DD_API_KEY}" -H 
+"dd-application-key:${PRIMARY_DD_APP_KEY}" --data "${CONNECTION}" --request POST ${PRIMARY_DD_API_URL}/api/v2/hamr
+```
+{{% /collapse-content %}} <br>
+
+
+
+#### when you have linked the DDR org to your primary org
+{{% collapse-content title=" 5. Create your Datadog API and App key for syncing" level="h5" %}}
+At the secondary Datadog site, create a set of `API key` **and** `App key`. You will use these keys in _steps 7_ to copy dashboards and monitors between Datadog sites. 
+
+<div class="alert alert-info">
+For your Agents, Datadog can copy API key signatures to the secondary backup account for you to prevent you from maintaining another set of API keys for your Agent. Contact your [Customer Success Manager](mailto:success@datadoghq.com) for any questions regarding this.
+</div>
+{{% /collapse-content %}}
+
+
+{{% collapse-content title=" 6. Configure Single Sign On for the Datadog App" level="h5" %}}
+Go to your [Organization Settings][1] to configure SAML or Google Login for your users. 
+
+**Single Sign On (SSO) is highly recommended** to enable all your users to be able to seamlessly login to your Disater Recovery organization during an outage. 
+
+You must invite your users to your Disaster Recovery organization and give them appropriate roles and permissions. Alternatively, to streamline this operation you can use [Just-in-Time provisioning with SAML][2].
```

Comment:
```suggestion
You must invite your users to your Disaster Recovery organization and give them appropriate roles and permissions. Alternatively, to streamline this operation, you can use [Just-in-Time provisioning with SAML][2].
```
What are "appropriate roles and permissions," and how would a user know how to provision them? What's JIT provisioning an alternative to?